scapy/utils.py

```diff
@@ -824,14 +832,15 @@ def read_packet(self, size=MTU):
 
     def read_block_idb(self, block, _):
         """Interface Description Block"""
+        # We should read options to set if_tsresol
```

Comment:
I don't really understand what "We should" means...

From what i understand, it's a feature that needs to be added in the future, right ? If so, it gonna be lost forever in that piece of code...